features/step_definitions/advocate_claim_steps.rb

```diff
@@ -91,16 +92,31 @@
 When(/^I add a fixed fee '(.*?)'$/) do |name|
   @claim_form_page.add_fixed_fee_if_required
   @claim_form_page.fixed_fees.last.select_fee_type name
+  wait_for_ajax
   @claim_form_page.fixed_fees.last.quantity.set 1
-  @claim_form_page.fixed_fees.last.rate.set "12.34"
+  wait_for_ajax
 end
 
 Then(/^I add a fixed fee '(.*?)' with case numbers$/) do |name|
   @claim_form_page.add_fixed_fee_if_required
   @claim_form_page.fixed_fees.last.select_fee_type name
-  @claim_form_page.fixed_fees.last.quantity.set 1
-  @claim_form_page.fixed_fees.last.rate.set "10.00"
+  wait_for_ajax
   @claim_form_page.fixed_fees.last.case_numbers.set "T20170001"
+  @claim_form_page.fixed_fees.last.quantity.set 1
+  wait_for_ajax
+end
+
+Then(/^I add a fixed fee '(.*?)' with defendants/) do |name|
```

Comment:
**[nit pick/not a blocker]** Is `with defendants` required for the description? If so, I can't see anything in the actual implementation suggesting we're doing something specific with defendants. 😕